doc/observability-deployment.adoc

```diff
@@ -188,6 +187,38 @@ spec:
     initialDelaySeconds: 30
     periodSeconds: 2
     timeoutSeconds: 10
+  startupProbe:
+    failureThreshold: 12
+    httpGet:
+      path: /health/started
+      port: 9443
+      scheme: HTTPS
+    initialDelaySeconds: 30
+    periodSeconds: 2
+    timeoutSeconds: 10
+...
+----
```

Comment:
I see you have the startupProbe section for the yaml below in the mpHealth-3.1+ section -- did you mean for it to be here as well?

Reply:
nope, I'll remove that